config/default/jenkins-infra.yaml

```diff
@@ -119,6 +124,24 @@ jenkins:
                           }
                         }
                       }
+                  - script: >
+                      multibranchPipelineJob('pluginsite') {
+                        displayName "Plugin Site"
+                        branchSources {
+                          github {
+                            id('2019081603')
+                            scanCredentialsId('github-access-token')
+                            repoOwner('jenkins-infra')
+                            repository('plugin-site')
+                            includes('master')
+                          }
+                        }
+                        factory {
+                          workflowBranchProjectFactory {
+                            scriptPath('Jenkinsfile')
```

Comment:
Thinking about this we might want a different jenkinsfile so it doesn’t get picked up by the other Jenkins instanced that don’t support k8s?

I’m off to bed now @halkeye Feel free to merge / amend if you want to tinker